Bottom: a069666dc428f246f9dc954bdb50553367794189
Top:    7553b4a2424ee63be0ec596d9d62a0f8512d235c
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-18 15:34:05 -0700

Refresh of rs-fulltest

---

diff --git a/man/rstream.1 b/man/rstream.1
index 58bd7e3..0909932 100644
--- a/man/rstream.1
+++ b/man/rstream.1
@@ -30,7 +30,8 @@ The number of messages to transfer from the client to the server and
 back again on each iteration.  (default 1)
 .TP
 \-S transfer_size
-The size of each send transfer, in bytes.  (default 1000)
+The size of each send transfer, in bytes.  (default 1000)  If 'all'
+is specified, rstream will run a series of tests of various sizes.
 .TP
 \-p server_port
 The server's port number.
